### DISCUSSION TITLE
Increase length of Custom TLV buffer

### DISCUSSION THREAD

**Discussion by gadrenayan** [lldpd/lldpd#598] [Q&A] [answered]
#define LLDP_TLV_ORG_OUI_INFO_MAXLEN 507

THis is the current limit of custom TLV buffer, can this be increase to say 1536, without affecting any other code.
Or will it require to change elsewhere as well.

**Comment by vincentbernat** [comment, marked as answer]
LLDP has a maximum length of 511 for a TLV. So, it is not possible to increase this limit.

**Reply by gadrenayan** [reply]
Ok, the length field itself is 9 bits, so we can't increase payload size beyond 511 bytes.
